data/data-pipeline/data_pipeline/etl/sources/fsf_flood_risk/etl.py

```diff
@@ -0,0 +1,93 @@
+# pylint: disable=unsubscriptable-object
```

Comment:
Note (namely to @lucasmbrown-usds) -- these two datasets get munged basically identically. We could move the work into a separate util function for transform, but the yaml file means that we wouldn't really save any lines of code. LMK if you think this explicit work is the wrong call.